documentation/example-file-usage/mqtt-example-file-usage.md

```diff
@@ -0,0 +1,136 @@
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command:
+
+   ```bash
+    python3 extract-auth-token.py -a
+   ```
+
+   You should get back a long sequence of numbers and letters, that's your Auth Token. Copy that to the clipboard for use in the next section.
+
+
+
+## Connecting to your M5 over MQTT
+
+Now here comes the fun part, we have everything we need to actually connect to the printer. It uses the [MQTT standard](https://mqtt.org/), to communicate with the wider world. It is possible to use this standard coupled with this repository build a tool that can remotely monitor the printer, print files, execute movement commands, pre-heat the printer, initiate the Auto-Bed Leveling routine, and just about anything else the companion phone app can. In this example, we'll simply be monitoring the printer. We can also save the output to a log file if desired.
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command, but replace "YOUR_AUTH_TOKEN_HERE" with your actual Auth Token and no quotes:
+
+   ```bash
+   python3 mqtt-connect.py -r us -A "YOUR_AUTH_TOKEN_HERE"
+   ```
+
+3. **[Optional Step]** If desired, you can save the contents of the output to a log file by adding `> output.log` to the end of the command in the previous step:
+
+   ```bash
+   python3 mqtt-connect.py -r us -A "YOUR_AUTH_TOKEN_HERE" > output.log
+   ```
+
+Now you should see a bunch of data updating in your terminal, this happens in 1 second intervals. It should look something like this and will be quite a bit more colorful:
+
+```
+TOPIC [/phone/maker/YOUR_SERIAL_NUMBER_HERE/notice]
+4d41c1000501020546c00100045b126436353163383633612d366266652d313033622d396234372d35623534393334336637356100000000000000000000000080d48d67e412394ccc2ef9d76b966687e047c862d36ca291d70a7c732aec8f28e7a315dc1dab0fc51eed678bee3959ae14af8ef3670553412e13cc90a0a6d2c4c0a949f072a716ef9153eed115eb7a7decf9c88bcb07922bae5cc925a96e954b1f70dfb55b079b696178f2c918c0af5c9e5861ae7809b97b80614cec6e948f86cc
+MqttMsg(
+    size=193,
+    m3=5,
+    m4=1,
+    m5=2,
+    m6=5,
+    m7=70,
+    packet_type=<MqttPktType.Single: 192>,
+    packet_num=1,
+    time=1678924548,
+    device_guid='SOME_DATA_HERE',
+    data=b'[{"commandType":1003,"currentTemp":5263,"targetTemp":18000},{"comman
+dType":1004,"currentTemp":3990,"targetTemp":6000}]'
+)
+```
+
+There's quite a bit to see, but let's just talk about the more interesting data from a layman's perspective  here:
+
+The TOPIC (that giant wall of numbers and letters at the top) is a string of data that represents what the printer is doing right now (homing, pre-heating, probing, etc.). When building your program, you could translate these messages to meaningful status messages for display.
```

Comment:
Sorry, that's wrong.

The "TOPIC" is the part in brackets:

`/phone/maker/YOUR_SERIAL_NUMBER_HERE/notice`

Think of it as the mqtt "path" that the message happens on. It's essentially a named queue of messages.

The hex string (`4d41c1000501020546c0010....`) is the raw payload bytes.

Decrypting and decoding those bytes yield the structured message (`MqttMsg(size=193, ... `)